### PR TITLE
Preparation for 1.0 release.

### PR DESCRIPTION
**Pull request by anpolimus**

### CODE REVIEW

File: modules/openy_gc_demo/migrations/migrate_plus.migration.virtual_y_node_blog.yml [hunk 1]
```diff
@@ -93,6 +96,14 @@ process:
     - plugin: migration
       migration: virtual_y_media_image
       source: image
+  field_gc_video_category:
+    -
+      plugin: migration
```
Comment (podarok) [inline modules/openy_gc_demo/migrations/migrate_plus.migration.virtual_y_node_blog.yml:101]:
```suggestion
      plugin: migration_lookup
```
see https://www.drupal.org/node/2861226

**Review by podarok — CHANGES_REQUESTED** [review]
Should be changed everywhere, please

File: modules/openy_gc_storage/openy_gc_storage.install [hunk 2]
```diff
@@ -75,3 +70,57 @@ function openy_gc_storage_update_8003() {
...
+  // Update configuration.
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'core.entity_form_display.node.vy_blog_post.default',
+    'core.entity_view_display.node.vy_blog_post.default',
+    'core.entity_view_display.node.vy_blog_post.teaser',
+    'field.field.node.vy_blog_post.field_gc_video_category',
+    'field.storage.eventinstance.field_ls_category',
+    'field.storage.eventseries.field_ls_category',
```
Comment (podarok) [inline modules/openy_gc_storage/openy_gc_storage.install:89]: not enough for storage configs. You need to use a different code here to have fully supported storage updates
See https://github.com/ymcatwincities/openy/pull/2215/files#diff-b205ecc1e2f71f78ba8f3db948e7cd27R108

**Review by podarok — CHANGES_REQUESTED** [review]

File: modules/openy_gc_shared_content/src/Controller/PreviewController.php [hunk 3]
```diff
@@ -45,7 +45,7 @@ public function openPreviewModal(Request $request, SharedContentSourceServerInte
     $response = new AjaxResponse();
     $instance = $this->sharedSourceTypeManager->createInstance($type);
     $query_args = $instance->getFullJsonApiQueryArgs();
-    $data = $instance->jsonApiCall($shared_content_source_server->getUrl(), $query_args, $uuid);
+    $data = $instance->jsonApiCall($shared_content_source_server, $query_args, $uuid);
```
Comment (podarok) [inline modules/openy_gc_shared_content/src/Controller/PreviewController.php:48]: looks like this was a bug

**Review by podarok — CHANGES_REQUESTED** [review]

File: js/gated-content/src/components/LogoutLink.vue [hunk 4]
```diff
@@ -1,17 +1,5 @@
 <template>
   <span class="menu">
-    <a @click.prevent="logout" href="#">Sign out</a>
+    <a href="/user/logout">Sign out</a>
```
[the remark this answers is not in the document]
Comment (podarok) [reply js/gated-content/src/components/LogoutLink.vue:3]: not a blocker, but still, would be nice to QA

File: modules/openy_gc_auth/src/GCUserAuthorizer.php [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    // Create drupal user if it doesn't exist and login it.
+    $account = user_load_by_mail($email);
+
+    if (!$account) {
+      $user = $this->userStorage->create();
+      $user->setPassword(user_password());
+      $user->enforceIsNew();
+      $user->setEmail($email);
+      $user->setUsername($name);
+      $user->addRole('virtual_y');
```
[the remark this answers is not in the document]
Comment (anpolimus) [reply modules/openy_gc_auth/src/GCUserAuthorizer.php]: done

File: js/gated-content/src/components/LogoutLink.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (hamrant) [reply js/gated-content/src/components/LogoutLink.vue:3]: Fixed - https://github.com/fivejars/openy_gated_content/pull/135

**Pull request merged**